File: inf1000/uke1/Hoyde.java
/**
1.13: Hoyde
*/

import java.util.Scanner;

class Hoyde {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        double lavHoyde = 140;
        double hoyHoyde = 190;

        System.out.print("Skriv inn din hoyde (cm): ");
        double minHoyde = Double.parseDouble(in.next());

        if (minHoyde < lavHoyde) {
            System.out.println("Du er lav.");
        } else if (minHoyde > hoyHoyde) {
            System.out.println("Du er hoy.");
        } else {
            System.out.println("Du er gjennomsnittelig.");
        }
    }
}

File: inf1001/uke2/hoyde.py
# Oppgave 2.11: Hoyde

min_hoyde = 140 #cm
max_hoyde = 190
egen_hoyde = input("Skriv inn din hoyde i cm: ")

if (egen_hoyde < min_hoyde):
    print "Du er lav."
elif (egen_hoyde > max_hoyde):
    print "Du er hoy."
else:
    print "Din hoyde er gjennomsnittlig."
